hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/statemachine/background/BlockDeletingService.java

```diff
@@ -541,10 +541,11 @@ private ContainerBackgroundTaskResult deleteViaTransactionStore(
      * Delete the chunks for the given blocks.
      * Return the deletedBlocks count and number of bytes released.
      */
-    private Pair<Integer, Long> deleteTransactions(
+    private DeleteTransactionStats deleteTransactions(
         List<DeletedBlocksTransaction> delBlocks, Handler handler,
         Table<String, BlockData> blockDataTable, Container container)
         throws IOException {
+      int blocksMissed = 0;
```

Comment:
Rename to `blocksProcessed` and increment it in the loop
`for (Long blkLong : entry.getLocalIDList()) {`

Reply:
Done~